storage/transfermanager/integration_test.go

```diff
@@ -213,114 +646,185 @@ func (tw *testWriter) WriteAt(b []byte, offset int64) (n int, err error) {
 	return len(b), nil
 }
 
-func TestIntegration_DownloadShard(t *testing.T) {
+func deleteExpiredBuckets(prefix string) error {
```

Comment:
We should look into creating an internal shared testutil for storage where these can live and be shared between the root package and any subpackages (Dataflux could also use these). Doesn't need to be in this PR though.